Api/Data/DeliveryOptionsInRestApi.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: richardperdaan
+ * Date: 2019-03-05
+ * Time: 14:41
+ */
+
+namespace MyParcelNL\Magento\Api\Data;
+
+interface DeliveryOptionsInRestApi
```

Comment:
DeliveryOptionsInRestApi
`In` is meer een omschrijvende vorm. Dit zou je opzich wel in een method verwachten, maar het voelt niet goed om dit als een class name te hebben.
DeliveryOptions is goed genoeg (want het zit al in de dir met Api\Data)

Reply:
Done